### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by emilysellers**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -12,12 +12,12 @@ const friendsSection = document.getElementById('friends-section');
...
 /* State */
 let message = '';
-let mushrooms = [{ type: 'porcini' }, { type: 'chanterelle' }, { type: 'morel' }];
+let mushrooms = [];
 
 let friends = [
     { name: 'Wilbur', satisfied: 0 },
     { name: 'Miss Piggy', satisfied: 0 },
-    { name: 'Pumbaa', satisfied: 0 },
+    { name: 'Pumbaa', satisfied: 3 },
```
Comment (aidanliddiard) [inline app.js]: I think you want satisfied: 0 here

File: app.js [hunk 2]
```diff
@@ -85,7 +85,10 @@ sayGoodbyeButton.addEventListener('click', () => {
     const stillHungry = [];
     for (const friend of friends) {
         // > if the friend is not fully satisfied, push
-        // them into the stillHungry array
+        if (friend.satisfaction < 3) {
```
Comment (aidanliddiard) [inline app.js]: This logic is not working because when you click the goodbye button all the friends disappear (not just the full ones). Do you see why?

**Review by aidanliddiard — COMMENTED** [review]
Just a few little changes I commented about

File: app.js (hunk 1 above)
Comment (emilysellers) [reply app.js]: Ah, yes, I'll correct that.

File: app.js (hunk 2 above)
Comment (emilysellers) [reply app.js]: I see it! I'll correct that now.

**Pull request merged**